Added the README in English language (#65)
cloud-barista/cb-coffeehouse: README.en.md

# Cloud-Barista's Coffeehouse

Explanation of different perspectives lowers the entry barriers for future contributors. 😉

Read this in other languages: [English](https://github.com/cloud-barista/cb-coffeehouse/blob/main/README.en.md) or [한국어](https://github.com/cloud-barista/cb-coffeehouse/blob/main/README.md).

> The historian Brian Cowan describes English coffeehouses as "places where people gathered to drink coffee, learn the news of the day, and perhaps to meet with other local residents and discuss matters of mutual concern.“ (See [English coffeehouses in the 17th and 18th centuries](https://en.wikipedia.org/wiki/English_coffeehouses_in_the_17th_and_18th_centuries))

English Coffeehouse (aka Penny House) was a social club where intellectuals shared various opinions while sipping a cup of coffee for 1 penny.

I hope that Cloud-Barista’s Coffeehouse can share various information “easily” like this.

> **<ins>Participate in open source projects by freely sharing information, explanations, and opinions, naturally find contribution points in Cloud-Barista's various Repositories, and become a Contributor, Reviewer, and Maintainer!**</ins> 😆

Reviewer: ```suggestion
**<ins>Cloud-Barista community encourages you to freely share information, comments, and opinions. This may be the first step towards getting involved in an open source project. In this process, you could find contribution points in Cloud-Barista's various repositories, and become a Contributor, Reviewer, and Maintainer naturally !**</ins> 😆
```

> Expect a lot of clarification/information, including the example below. 😊

Reviewer: ```suggestion
We expect lots of explanation/information, including the example below. 😊
```

> - Cloud or multi-cloud concept
> - Cloud-Barista related terms, technologies, and concepts
>   - Microservice architecture (MSA)
>   - gRPC (Remote Procedure Call)
>   - GitHub Container Registry (GHCR)
>   - GitHub Actions
>   - Golang CI/CD (Continuous Integration/Continuous Delivery)
> - Cloud-Barista Issues and Solutions
> - Installation/Deployment Automation Script

Reviewer: ```suggestion
- Cloud or Multi-cloud concept
- Cloud-Barista related terms, technologies, and concepts
  - Microservice architecture (MSA)
  - gRPC (Remote Procedure Call)
  - GitHub Container Registry (GHCR)
  - GitHub Actions
  - Golang CI/CD (Continuous Integration/Continuous Delivery)
- Cloud-Barista issues and those solutions
- Installation/deployment automation scripts
```

An easy explanation from a beginner's point of view will be of great help to future Cloud-Barista Contributors. 👍

🎁 [Message for future contributors](https://github.com/cb-contributhon/cb-coffeehouse/wiki/Interview-with-Seokho-Son) 💯 😍 ⭐ 👍

> ## Description/information sharing method (draft)

Reviewer: ```suggestion
## Description/information sharing guide
```

> The method below is somewhat of a guide.

Reviewer: ```suggestion
This is a rough guide to mitigating unnecessary redundant contributions.
```
